wye/workshops/models.py

```diff
@@ -51,6 +56,35 @@ class Meta:
     def __str__(self):
         return '{}-{}'.format(self.requester, self.workshop_section)
 
+    def save(self, force_insert=False, force_update=False, using=None):
+        super(Workshop, self).save(force_insert, force_update, using)
```

Comment:
That exactly problem If we also update it keep sending email right ?
We need to send email on new instance creation ?